noether_conversions/src/noether_conversions.cpp

```diff
@@ -249,48 +241,40 @@ visualization_msgs::MarkerArray convertToAxisMarkers(const std::vector<geometry_
     marker.points.push_back(p2);
   };
 
-  for(auto& poses : path)
+  for (const auto& tool_path : toolpaths.paths)
```

Comment:
I'd like to suggest not using auto. It is convenient to the person writing the code, but makes later development more difficult, because someone trying to dive in and understand the code has to decipher what kind of an object each thing declared as auto really is. In this case, toolpaths was a noether_msgs item, then segment was some item from that. The rabbit hole gets deep. Perhaps I'm just not up with the times, but it seems just lazy to me.

Reply:
Fixed.